src/history/base.js

```diff
@@ -139,10 +139,17 @@ export class History {
     const prev = this.current
     this.current = route
     this.cb && this.cb(route)
+    this.updateTitle(route)
     this.router.afterHooks.forEach(hook => {
       hook && hook(route, prev)
     })
   }
+
+  updateTitle (route: Route) {
+    if (route.title !== '') {
+      window.document.title = route.title
```

Comment:
my code is not good       

dose there need a judge for window ?  or how to solve  this?  can you help me?